config/_default/config.toml

```diff
@@ -93,7 +93,7 @@ weight = 30
 [[languages.en.menu.main]]
 identifier = "documentation-core"
 name = "Core"
-url = "https://kokkos.github.io/kokkos-core-wiki/"
+url = "https://kokkos.github.io/kokkos-core-documentation/"
```

Comment:
this is not how this works. This will just point to a non-existing website we need to figure out how to introduce aliases for github sites which don't just point to an existing repo